.github/workflows/repository-scanner.yml

```
@@ -0,0 +1,29 @@
+name: Scan Actions
+on:
+  workflow_call:
+  push:
+
+jobs:
+  action-allowedlist:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+        with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-Actions
+          path: Ed-Fi-Actions
+      - name: Checkout
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+        with:
+          path: testing-repo
+      - name: Scan used actions
+        uses: ./Ed-Fi-Actions/action-allowedlist@latest
+        id: scan-action
+  bidi-scanner:
+      runs-on: ubuntu-latest
+      steps:
+        # Check out the repo that is calling this action
+      - name: Checkout Repo to Scan
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+      - name: Trojan Source Detector
+        uses: haveyoudebuggedit/trojansourcedetector@103b76ee743f7a666fd98dfaa15b4b7e314ca140 #Hash value for v1.1.0
```

Why have two different jobs, instead of just another step after "scan used actions"?

```suggestion
      - name: Scan used actions
        uses: ./Ed-Fi-Actions/action-allowedlist@latest
        id: scan-action
      - name: Trojan Source Detector
        uses: haveyoudebuggedit/trojansourcedetector@103b76ee743f7a666fd98dfaa15b4b7e314ca140 #Hash value for v1.1.0
```